examples/conditional_statements/conditional_statements.md

```diff
@@ -38,13 +60,15 @@ ashia_age := 38
 
 if tom_age < ashia_age {
   if tom_age < 18 {
-	println('tom_age < 18 and younger than Ashia.')
-  } else {
-	println('tom_age > 18 and older than Ashia.')
-  }
-} else {
-	println('$tom_age == $ashia_age')
-	println('Tom and Ashia have the same age.')
+     println('tom_age < 18 and younger than Ashia.')
+    } else {
+      println('tom_age >= 18 and younger than Ashia.')
+     }
+   } 
+   else if tom_age > ashia_age {
+     println('$tom_age > $ashia_age')
+   } else {
+     println('$tom_age == $ashia_age')
```

Comment:
This identation is not similar to the other ones in this PR. They should be the same to keep consistency.